Optimizer/IslandsOptimizer/IslandsOptimizer.h

```diff
@@ -0,0 +1,210 @@
+//  MABE is a product of The Hintze Lab @ MSU
+//     for general research information:
+//         hintzelab.msu.edu
+//     for MABE documentation:
+//         github.com/Hintzelab/MABE/wiki
+//
+//  Copyright (c) 2015 Michigan State University. All rights reserved.
+//     to view the full license, visit:
+//         github.com/Hintzelab/MABE/wiki/License
+
+#pragma once
+
+#include "../AbstractOptimizer.h"
+#include "../../Utilities/MTree.h"
+
+#include <iostream>
+#include <sstream>
+
+
+class IslandsOptimizer : public AbstractOptimizer {
+public:
+
+	static std::shared_ptr<ParameterLink<std::string>> IslandNameSpaceListPL;
+	static std::shared_ptr<ParameterLink<double>> migrationRatePL;
+
+	std::vector <std::shared_ptr<AbstractOptimizer>> islandOptimizers;
+	int islands;
```

Comment:
Just make this size_t. Thenyou can avoid the static_cast in the .cpp

Reply:
I changed this to size_t, but I still needed one static cast when writing to data map.